src/panels/lovelace/cards/hui-light-card.ts

```diff
@@ -0,0 +1,256 @@
+                @click="${this._handleClick}"
+              ></ha-icon>
+              <div class="brightness">${Math.round(
+                (stateObj.attributes.brightness / 254) * 100
+              ) || 0}%</div>
+              <div class="name">${this._config.name ||
+                computeStateName(stateObj)}</div>
+            </div>
+          </div>
+        </div>
+      </ha-card>
+    `;
+  }
+
+  protected shouldUpdate(changedProps) {
+    if (changedProps.get("hass")) {
+      return (
+        changedProps.get("hass").states[this._config!.entity] !==
+        this.hass!.states[this._config!.entity]
+      );
+    }
+    if (changedProps.get("_config")) {
+      return changedProps.get("_config") !== this._config;
+    }
+    return changedProps;
+  }
+
+  protected firstUpdated() {
+    jQuery("#light", this.shadowRoot).roundSlider({
+      ...lightConfig,
+      change: (value) => this._setBrightness(value),
+      drag: (value) => this._dragEvent(value),
+    });
+  }
+
+  protected updated() {
+    const attrs = this.hass!.states[this._config!.entity].attributes;
+
+    jQuery("#light", this.shadowRoot).roundSlider({
+      value: Math.round((attrs.brightness / 254) * 100) || 0,
+    });
+  }
+
+  private renderStyle() {
+    return html`
+    ${roundSliderStyle}
+    <style>
+      :host {
+        display: block;
+      }
+      ha-card {
+        overflow: hidden;
+      }
+      #root {
+        position: relative;
+        overflow: hidden;
+      }
+      #tooltip {
+        position: absolute;
+        top: 0;
+        left: 0;
+        right: 0;
+        height: 100%;
+        text-align: center;
+        z-index: 15;
+      }
+      .icon-state {
+        display: block;
+        margin: auto;
+        width: 100%;
+        height: 100%;
+        transform: translate(0,25%);
+      }
+      #light {
+        margin: 0 auto;
+        padding-top: 16px;
+        padding-bottom: 8px;
+      }
+      #light .rs-bar.rs-transition.rs-first, .rs-bar.rs-transition.rs-second{
+        z-index: 20 !important;
+      }
+      #light .rs-range-color  {
+        background-color: var(--primary-color);
+      }
+      #light .rs-path-color  {
+          background-color: #d6d6d6;
+      }
+      #light .rs-handle  {
+          background-color: #FFF;
+          padding: 7px;
+          border: 2px solid #d6d6d6;
+      }
+      #light .rs-handle.rs-focus  {
+          border-color:var(--primary-color);
+      }
+      #light .rs-handle:after  {
+          border-color: var(--primary-color);
+          background-color: var(--primary-color);
+      }
+      #light .rs-border  {
+        border-color: transparent;
+      }
+      ha-icon {
+        display: flex;
+        margin: auto;
+        width: 40%;
+        height: 40%;
+        color: var(--paper-item-icon-color, #44739e);
+        cursor: pointer;
+      }
+      ha-icon[data-domain=light][data-state=on],
+      ha-icon[data-domain=switch][data-state=on],
```

Comment:
what

Reply:
Ah yea I copied the button card icon styling. oops